models/jira.yml

```diff
@@ -59,20 +62,20 @@ models:
       - name: is_current_record
         description: Boolean indicating if this is the current/active field state.
       - name: issue_timestamp_id
-        description: Surrogate key hashed on `valid_from` and `issue_id`
+        description: Surrogate key hashed on `valid_from`, `issue_id`, and `source_relation`
         tests:
-          - unique
           - not_null
         # other fields included in var(issue_history_columns)
 
   - name: jira__daily_sprint_issue_history
     description: >
       Each record represents a snapshot of a sprint and its assorted issues on a given day between the sprint start date and the most recent update to the sprint.
     columns:
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
       - name: sprint_issue_day_id
-        description: Surrogate key hashed on `date_day`, `sprint_id`, and `issue_id`
+        description: Surrogate key hashed on `date_day`, `sprint_id`, `issue_id`, and `source_relation`
```

Comment:
same request to add the test back in and document the surrogate key change in the changelog

Reply:
updated

Reply:
Ugh. Updated.